README.md

```diff
@@ -130,9 +130,27 @@ Variable | Description | Example
 --- | --- | ---
 `ENABLE_AUTH` | Enable authentication | 1
```

Comment:
The internal authentication type is the only type that can work pout of the box, that's why it's the default. I'll give the idea of removing the flag and inferring it from an empty AUTH_TYPE some thought, it may simplify things!